src/main/java/com/samoilenko/onlinebookstore/mapper/UserMapper.java

```diff
@@ -0,0 +1,21 @@
+package com.samoilenko.onlinebookstore.mapper;
+
+import com.samoilenko.onlinebookstore.config.MapperConfig;
+import com.samoilenko.onlinebookstore.dto.UserDto;
+import com.samoilenko.onlinebookstore.dto.UserRegistrationRequestDto;
+import com.samoilenko.onlinebookstore.dto.UserResponseDto;
+import com.samoilenko.onlinebookstore.model.User;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+
+@Mapper(config = MapperConfig.class)
+public interface UserMapper {
+
+    UserDto toDto(User user);
+
+    UserResponseDto toResponseDto(User user);
```

Comment:
no need to have 2 methods here, because we do not need the `UserDto` with a password in it, let's leave only `UserResponseDto` and a method for it